### ISSUE TITLE
Bug Report: Incorrect workflow files

### ISSUE THREAD

**Issue by EshaanAgg** [PalisadoesFoundation/talawa-api#1452]
**Describe the bug**
In the [pull request workflow](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/.github/workflows/pull-request.yml), we currently are using a GitHub action to get all the changes files with respect to the PR, and then running the various jobs (type-check, lint-check, format-check) on the same which is not correct, as this leads to these jobs wither being performed on files of incorrect extensions, or the files which must be ignored (specified in various `.config` files).  This must be changed. 

**To Reproduce**
Check the [log of the recent PR](https://github.com/PalisadoesFoundation/talawa-api/actions/runs/7104785014/workflow?pr=1451).

**Expected behavior**
The lining and formatting actions should be performed on the whole codebase as earlier, or they should only be run for the files with proper extensions and configurations. How the same is to be done requires research. 

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by skbhagat0502** [comment]
Hey @EshaanAgg, I noticed my recent PR on the pull request workflow got merged, but it seems there's the bug that cropped up as a result. Apologies if it wasn't up to your expectations. I'd like to resolve this issue.

**Comment by skbhagat0502** [comment]
Should I make it as before so that linting formating and type checking would run for all the files or they should run for the modified files only as my PR but for files with proper extension and configurations.

**Comment by EshaanAgg** [comment]
@skbhagat0502 The later is more efficient, but ensuring the correctness of the same would be a challenge. Please proceed with whichever you feel would be the best suited.

**Comment by skbhagat0502** [comment]
@EshaanAgg I will do my best research to resolve this issue with the later approach.

**Comment by skbhagat0502** [comment]
Ok @EshaanAgg

**Comment by skbhagat0502** [comment]
@EshaanAgg is this issue resolved because I can see the pullrequest.yml file is now as before.

**Comment by EshaanAgg** [comment]
Yes. I had to do it cause it was causing a lot of PRs to fail. You can now work on coming up with a per-file solution if you wish.

**Comment by Cioppolo14** [comment]
Closing this as it appears to be fixed based on the discussion above.
